app/portainer/views/stacks/create/createStackController.js

```diff
@@ -104,13 +90,9 @@ function ($scope, $state, StackService, Authentication, Notifications, FormValid
       return;
     }
 
-    var type = $scope.state.StackType;
```

Comment:
As specified in the comments in the `StackService`, I'd rather keep the old behavior here. Can you revert the changes?